src/app/pages/poll/poll.component.ts

```diff
@@ -100,6 +101,25 @@ export class PollComponent {
     });
   }
 
+  /**
+   * Update the current poll item for this poll on the server
+   *
+   * @param event Click event
+   * @param pollItemId Id of the poll item
+   */
+  onRunPollItem(event: any, pollItemId: number): void {
```

Comment:
You mean apart from the type `any`?

Reply:
yeah, exactly

Reply:
I looked it up and it seems like untyped event passing is indeed a thing in Angular, since it even is listed in the official documentation as valid way of doing this. However, I also found and implemented an event class (`MouseEvent`) to type this more strict.